saleor/graphql/channel/mutations/channel_create.py

```diff
@@ -24,6 +30,20 @@ class StockSettingsInput(graphene.InputObjectType):
     )
 
 
+class OrderSettingsInput(graphene.InputObjectType):
+    automatically_confirm_all_new_orders = graphene.Boolean(
+        required=False,
+        description="When disabled, all new orders from checkout "
+        "will be marked as unconfirmed. When enabled orders from checkout will "
+        "become unfulfilled immediately.",
```

Comment:
Would be worth to mention that by default it will be set to `True`/`False`.
Same for `automatically_fulfill_non_shippable_gift_card`